src/07-embedding/02-artists.ts

```diff
@@ -0,0 +1,151 @@
+import { ARTIST_IDS } from "./artist-ids"
+import weaviate from "weaviate-ts-client"
+import _ from "lodash"
+
+const client = weaviate.client({
+  scheme: "https",
+  host: "https://weaviate.stg.artsy.systems",
+})
+
+const CLASS_NAME: string = "ArtistBio2"
+
+const BATCH_SIZE = 100
+
+async function main() {
+  const artists = await fetchArtists()
+  await prepareArtistsCollection()
+  await insertArtists(artists)
+  /*
+   * now query from Weaviate console:
+   * https://link.weaviate.io/3UEZ854
+   */
+}
+
+main()
+
+async function prepareArtistsCollection() {
+  const client = weaviate.client({
+    scheme: "https",
+    host: "https://weaviate.stg.artsy.systems",
+  })
+
+  const alreadyExists = await client.schema.exists(CLASS_NAME)
+
+  if (alreadyExists) {
+    console.log(`${CLASS_NAME} class already exists, deleting it`)
+    await client.schema.classDeleter().withClassName(CLASS_NAME).do()
+  }
+
+  const classWithProps = {
+    class: CLASS_NAME,
+    vectorizer: "text2vec-openai",
+    moduleConfig: {
+      "text2vec-openai": {
+        model: "ada",
+        dimensions: 1536,
+        type: "text",
+        vectorizeClassName: false,
+      },
+    },
+    properties: [
+      {
+        name: "blurb",
+        dataType: ["text"],
+        moduleConfig: {
+          "text2vec-openai": {
+            skip: false,
+            vectorizePropertyName: false,
+          },
+        },
+      },
+      {
+        name: "slug",
+        dataType: ["text"],
+        moduleConfig: {
+          "text2vec-openai": {
+            skip: true,
```

Comment:
Individual fields can also be configured: re the vectorization. Here we skip vectorizing the slug, which doesn't really have any semantic content.